framework/db/ar/CActiveFinder.php

```diff
@@ -1262,7 +1262,7 @@ public function join($element)
 	public function createCommand($builder)
 	{
 		$sql=($this->distinct ? 'SELECT DISTINCT ':'SELECT ') . implode(', ',$this->selects);
-		$sql.=' FROM ' . implode(' ',$this->joins);
+		$sql.=' FROM ' . implode(' ',array_unique($this->joins));
```

Comment:
@beezee Pulled this fix from [this merged PR](https://github.com/yiisoft/yii/pull/2977/files) in the Yii repo. Not sure on Yii 2.0, that would be a beast of an upgrade. Might look into at least getting onto 1.1.16